.github/workflows/stale-bot.yml

```diff
@@ -0,0 +1,33 @@
+# Go to https://github.com/actions/stale for the full documentation of these options.
+
+name: 'Stale bot'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v3.0.18
+        with:
+          # Rate limit per run, defaults to 30.
+          operations-per-run: 30
+
+          # Only issues with both `reproduction:needed` and `type:bug` will be touched by the stale bot.
+          only-issue-labels: 'reproduction:needed,type:bug'
```

Comment:
Stale bot will:

- Only affect issues, and completely skip over any PRs
- Only touch issues with `reproduction:needed` AND `type:bug` labels